Doc/library/statistics.rst

```diff
@@ -830,8 +830,12 @@ of applications in statistics.
 
        .. versionchanged:: 3.13
```

Comment:
The existing documentation above this says of *seed* "this is useful for creation reproducible results, even in a multi-threading context." and the PR this is modifying mentions that `random.gauss()` is problematic describing the non-gauss version as better for being "more self contained and less vulnerable to concurrency issues (because gauss() is stateful between successive calls)."

I presume we might want call that implementation detail out with a sentence added to the above paragraph recommending the new *use_gauss=False* argument to people if that caveat matters to them?  @rhettinger